mleap-runtime/src/main/scala/ml/combust/mleap/runtime/transformer/Pipeline.scala

```diff
@@ -2,15 +2,15 @@ package ml.combust.mleap.runtime.transformer
 
 import ml.combust.mleap.core.Model
 import ml.combust.mleap.core.types.{DataType, NodeShape, StructField, StructType}
-import ml.combust.mleap.runtime.frame.{FrameBuilder, Transformer}
+import ml.combust.mleap.runtime.frame.{FrameBuilder, Transformer => FrameTransformer}
```

Comment:
is the alias needed?

Reply:
I don't think it is strictly necessary, but there was a compiler warning about covering up names or something.  

there is technically a `ml.combust.mleap.runtime.Transformer` class, which is literally defined as `type Transformer = frame.Transformer`.  So it was covering itself.  Doesn't impact correctness, but I was trying to clean up warnings because they bother me 😄

Reply:
ah i see, okay :D